File: loop-structures/RightAngledTriangle.js
const prompt = require("prompt-sync")();

let lengthOfBase = parseInt(prompt("Enter the length of base between (1-10): "))

while (lengthOfBase < 1 || lengthOfBase > 10) {
    lengthOfBase = parseInt(prompt("Enter the length of base between (1-10): "))
}

for (let index = 1; index <= lengthOfBase; index++) {
    for (let count = 1; count <= index; count++) {
        process.stdout.write("*");
    }
    console.log();
}

File: loop-structures/right_angled_triangle.py
length_of_base = int(input("Enter the length of base between (1-10): "))

while length_of_base < 1 or length_of_base > 10:
    length_of_base = int(input("Enter the length of base between (1-10): "))

for index in range(1,length_of_base +1):
    for count in range(1,index +1):
        print("*", end="")
    print()

